examples/helloworld/README.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Helloworld Example
+==================
+
+**Requires: python3**
+
+This example creates a two activities and one decider for those activities. The first activity will
+ask for a name and then the second activity will print that name.
+
+How To
+------
+
+Preconfiguration
```

Comment:
Totally agree! I think it makes the most sense to move this section into: https://github.com/boto/botoflow/blob/master/docs/source/setting_up_dev_env.rst Let me know what you think of the current commit